nebula_ros/schema/sub/lidar_hesai.json

```diff
@@ -32,6 +32,20 @@
       "maximum": 1200,
       "multipleOf": 60
     },
+    "sync_angle": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 359,
+      "default": "0",
+      "description": "The angle in whole degrees which should be hit on top-of-second."
+    },
+    "cut_angle": {
+      "type": "number",
+      "minimum": 0,
+      "maximum": 360,
```

Comment:
[Future work] in the documentation it may be worth explaining why the cut angle can go to 360, but sync angle can not. Sync angle is an integer and cut angle is a float would be sufficient, it's just a little confusing how the sensor and driver handles 360 vs 0.